config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -70,12 +70,12 @@ presubmits:
         - --
         - --deployment=node
         - --gcp-zone=us-west1-b
-        - '--node-test-args=--container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"'
+        - '--node-test-args=--container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"'
         - --node-tests=true
         - --provider=gce
         - --test_args=--nodes=8 --focus="\[NodeConformance\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"
         - --timeout=65m
-        - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/containerd/image-config.yaml
+        - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/containerd/image-config-systemd.yaml
```

Comment:
This is the blocking pull job (`pull-kubernetes-node-e2e-containerd`), and runs on every PR in k/k (`always_run: true`). Can we hold off on making the change to this job until we validate that the changes work in the other jobs? Then we can followup and make the change here as well. The goal is just to avoid distrupting the main blocking job.

Reply:
+1, very reasonable. @akhilerm can you please split this PR into two?

Reply:
Done.

Reply:
Have reverted the changes in `pull-kubernetes-node-e2e-containerd` for now. Will raise the other PR once the changes in this PR are validated to be working.